### ISSUE TITLE
如何覆盖tailwindcss的媒体查询配置信息

### ISSUE THREAD

**Issue by xuyifeiOnly** [unocss/unocss#2983]
### Clear and concise description of the problem

我现在使用uniocss，但是sm，md对应的媒体查询都是只有min-width,我希望是下面这样的
 screens: {
      'sm': {'min': '640px', 'max': '767px'},
      // => @media (min-width: 640px and max-width: 767px) { ... }

      'md': {'min': '768px', 'max': '1023px'},
      // => @media (min-width: 768px and max-width: 1023px) { ... }

      'lg': {'min': '1024px', 'max': '1279px'},
      // => @media (min-width: 1024px and max-width: 1279px) { ... }

      'xl': {'min': '1280px', 'max': '1535px'},
      // => @media (min-width: 1280px and max-width: 1535px) { ... }

      '2xl': {'min': '1536px'},
      // => @media (min-width: 1536px) { ... }
    }
我尝试了uniocss覆盖或者在tailwind.config.js中覆盖都不行，想知道这个有啥解决办法吗?

### Suggested solution

我现在使用uniocss，但是sm，md对应的媒体查询都是只有min-width,我希望是下面这样的
 screens: {
      'sm': {'min': '640px', 'max': '767px'},
      // => @media (min-width: 640px and max-width: 767px) { ... }

      'md': {'min': '768px', 'max': '1023px'},
      // => @media (min-width: 768px and max-width: 1023px) { ... }

      'lg': {'min': '1024px', 'max': '1279px'},
      // => @media (min-width: 1024px and max-width: 1279px) { ... }

      'xl': {'min': '1280px', 'max': '1535px'},
      // => @media (min-width: 1280px and max-width: 1535px) { ... }

      '2xl': {'min': '1536px'},
      // => @media (min-width: 1536px) { ... }
    }
我尝试了uniocss覆盖或者在tailwind.config.js中覆盖都不行，想知道这个有啥解决办法吗?

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/unocss/unocss/blob/main/CONTRIBUTING.md).
- [X] Read the `README.md` of using the package.
- [X] Already used the [Interactive Docs](https://unocss.dev/interactive/).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by zyyv** [comment]
Redefine the **UnoCSS** theme of `breakpoints` key, https://unocss.dev/config/theme#usage-in-rules

And try to use breakpoints variant like you want. https://unocss.dev/presets/wind#breakpoints

**Comment by zyyv** [comment]
> 非常感谢，才发现uniocss已经内部集成了

是 **UnoCSS**😵‍💫

**Comment by im-pan** [comment]
hello i have similar problem
I want to add a mm: or override sm: to make his decision as follows:
=> @media screen only and (hover:none) and (pointer:rough){}
What should I do?

**Comment by AAABingBing** [comment]
> [Playground](https://unocss.dev/play/?html=DwEwlgbgBA7gTgQwA5IKZwHwCgpVJKBAFwFoBbEALiNQA9S5UQoAbUgZzOrtICMWArqlYBzbvRIjGqAHYYAqjID2AYQDKa4AHpwEbNt3YgA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAJIBjAlT6UaMAKpYIcAL5x0UCCDgByNnOFUqGgFAHUAD0iwUGAIZsANvDSYc%2BQsCIAKBAbhwqAC2gYYQ4qAC5Ebx84AHcoKzAwVChwjWiAWgBGAAYsgDdouD9MnNy-ZVtTOGAYVBAqNOFULBqoOAArNipOHgamlo0SSIVBnxg-WtRwryi4FkorAGtIbBgwiJmfKhAUgGYAJiywEwHImZBkFIA2ABZD45GN2yIUgHYADjuTmeGhh-4pNYAbVO5EktFkEHcAEoHj5-rQGExWBwuNxobDQQIYCIxJ4QZthFYKuEMgA6PYYnzCZBYFJ%2BRhgMIAeiZ1BApP8TK%2BUQUMMiAF1BryDEA&css=Q&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g)

@zyyv This method is indeed perfect, but I did not find any case of this usage in the documentation. Is it possible to consider adding this method to the official documentation?

**Comment by zyyv** [comment]
> > [Playground](https://unocss.dev/play/?html=DwEwlgbgBA7gTgQwA5IKZwHwCgpVJKBAFwFoBbEALiNQA9S5UQoAbUgZzOrtICMWArqlYBzbvRIjGqAHYYAqjID2AYQDKa4AHpwEbNt3YgA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAJIBjAlT6UaMAKpYIcAL5x0UCCDgByNnOFUqGgFAHUAD0iwUGAIZsANvDSYc%2BQsCIAKBAbhwqAC2gYYQ4qAC5Ebx84AHcoKzAwVChwjWiAWgBGAAYsgDdouD9MnNy-ZVtTOGAYVBAqNOFULBqoOAArNipOHgamlo0SSIVBnxg-WtRwryi4FkorAGtIbBgwiJmfKhAUgGYAJiywEwHImZBkFIA2ABZD45GN2yIUgHYADjuTmeGhh-4pNYAbVO5EktFkEHcAEoHj5-rQGExWBwuNxobDQQIYCIxJ4QZthFYKuEMgA6PYYnzCZBYFJ%2BRhgMIAeiZ1BApP8TK%2BUQUMMiAF1BryDEA&css=Q&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g)
> 
> @zyyv This method is indeed perfect, but I did not find any case of this usage in the documentation. Is it possible to consider adding this method to the official documentation?

Of course, this is indeed different from tailwind, which I will add to the documentation later.
